Cross Region VM Restore Points/Cross Region VM Restore Points.md

```diff
@@ -0,0 +1,391 @@
+# Cross-Region-VM-Restore-Points
```

Comment:
Name the .md file as either ReadMe.md or "Cross Region Copy of VM Restore Points.md". First suggestion would ensure that future features can be added in the same ReadMe file, second suggestion is to have separate files for each feature.

Reply:
Done. Lets have separate files for each feature.